gennav/utils/graph.py

```diff
@@ -33,6 +33,32 @@ def add_edge(
         self.edges[to_node].append(from_node)
         self.distances[(from_node, to_node)] = self.calc_dist(from_node, to_node)
 
+    def del_edge(self, from_node, to_node):
+        """Deletes edge connecting two nodes to the graph.
+
+        Args:
+            from_node (gennav.utils.common.Node): starting node of the edge.
+            to_node (gennav.utils.common.Node): ending node of the edge.
+        """
+
+        if len(self.edges[from_node]):
```

Comment:
Can you make this if .. == 0:

Makes it more explicit

Reply:
Done.